src/test/resources/testbib/bibWithUserComments.bib

```diff
@@ -33,12 +33,12 @@ @Comment this
 @INPROCEEDINGS{1137631,
   author = {Gustav Bostr\"{o}m and Jaana W\"{a}yrynen and Marine Bod\'{e}n and
 	Konstantin Beznosov and Philippe Kruchten},
-  title = {Extending XP practices to support security requirements engineering},
```

Comment:
Why is here the `title` **after** `booktitle` and in `bibWithUserCommentAndEntryChange.bib`, the `title` **before** `booktitle` (which I like)

We had huge debates about field ordering. I think, we agreed that `author` comes first and then `title`. Then the other metadata.